lib/Gocdb_Services/Config.php

```diff
@@ -567,4 +567,59 @@ public function getEmailTo()
 
         return $emailTo;
     }
+
+    public function getIdentityProvidersInfo(): array
+    {
+        $localInfo = $this->GetLocalInfoXML();
+        $identityProviders = [];
+
+        if (!empty($localInfo->identity_providers->provider)) {
+            foreach (
+                $localInfo
+                    ->identity_providers
+                    ->provider as $providerDetails
+            ) {
```

Comment:
Something like this I think would read better

```suggestion
        $configured_providers = $localInfo->identity_providers->provider;
        if (!empty($configured_providers)) {
            foreach ($configured_providers as $providerDetails) {
```

Reply:
Done in 5091398904b06339ae9493205330b88a6b8faac2